libs/ha/ha_common_libs_k8s.py

```diff
@@ -743,7 +744,7 @@ def check_cluster_status(self, pod_obj, pod_list=None):
             resp = pod_obj.execute_cmd(common_cmd.CLSTR_STATUS_CMD.format(self.dir_path))
         except IOError as error:
             LOGGER.error("Error: Cluster status has some failures.")
-            LOGGER.debug("Response for cluster status: %s",resp)
+            LOGGER.debug("Response for cluster status: %s", error)
```

Comment:
This will always print cluster logs whenever cluster has offline value. We are already sending error in return, can we just not print as we are using this function in many test specially in loop for K pods down.

Reply:
I agree

Reply:
yes, i agree too.